server/tests/metadata.rs

```diff
@@ -302,6 +304,23 @@ mod entities_metadata_query {
         }));
     }
 
+    #[actix_rt::test]
+    async fn assert_that_query_is_faster_then_2000ms() {
```

Comment:
```suggestion
    async fn assert_query_is_faster_than_3000ms() {
```
Implementation of the test looks like 3s, not 2s.